src/Sylius/Bundle/ShopBundle/templates/shared/layout/base/header/top_bar/locale_switch.html.twig

```diff
@@ -1,14 +1,23 @@
 {% if available_locales|length > 1 %}
     <div class="col-auto">
-        <div {{ sylius_test_html_attribute('locale-selector') }}>
-            <span {{ sylius_test_html_attribute('active-locale') }}>{{ active_locale|sylius_locale_name }}</span>
-            {% for code in available_locales %}
-                {% if code is not same as(active_locale) %}
-                    <a href="{{ path('sylius_shop_switch_locale', {'code': code}) }}" {{ sylius_test_html_attribute('available-locale') }}>
-                        {{ code|sylius_locale_name }}
-                    </a>
-                {% endif %}
-            {% endfor %}
+        <div class="dropdown" {{ sylius_test_html_attribute('locale-selector') }}>
+            <button class="btn btn-sm btn-dark dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" {{ sylius_test_html_attribute('active-locale', active_locale) }}>
+                <span class="fi fi-{{ active_locale|sylius_locale_country|lower }} me-2"></span>
+                {{ active_locale|sylius_locale_country }}
```

Comment:
What gets shown in case of short locales like `en`?

Reply:
It wont show the flag and the code. But we will fix it in other task